projects/media-viewer/src/lib/store/effects/bookmarks.effects.ts

```diff
@@ -27,6 +35,21 @@ export class BookmarksEffects {
   createBookmark$ = this.actions$.pipe(
     ofType(bookmarksActions.CREATE_BOOKMARK),
     map((action: bookmarksActions.CreateBookmark) => action.payload),
+    withLatestFrom(
+      this.store.pipe(select(fromAnnotations.getDocumentIdSetId)),
+      this.store.pipe(select(fromBookmarks.getPdfPosition))
```

Comment:
u can't connect here store its wrong implementation of ngrx... pass the payoad through action

Reply:
I'm not sure why, I've used this pattern before, it's even used in the ngrx examples (https://ngrx.io/guide/effects, collections.effect)